api/TaskTracker/TaskTracker.Web/Program.cs

```diff
@@ -16,8 +36,18 @@
     app.UseSwaggerUI();
 }
 
+using (IServiceScope scope = app.Services.CreateScope())
+{
+    TaskTrackerDbContext dbContext = scope.ServiceProvider.GetRequiredService<TaskTrackerDbContext>();
+    UserManager<UserEntity> userManager = scope.ServiceProvider.GetRequiredService<UserManager<UserEntity>>();
+    RoleManager<RoleEntity> roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<RoleEntity>>();
+
+    dbContext.Database.Migrate();
+}
+
 app.UseHttpsRedirection();
 
+app.UseAuthentication();
```

Comment:
what does this line do?

Reply:
enable authentication for user access control

Reply:
but we don't use authentication for now, we just want to setup database